src/test/java/uk/gov/pay/adminusers/utils/DatabaseTestHelper.java

```diff
@@ -206,14 +207,14 @@ public DatabaseTestHelper addService(Service service, String... gatewayAccountId
             if (merchantDetails == null) {
                 merchantDetails = new MerchantDetails();
             }
-            return handle.createStatement("INSERT INTO services(" +
+            return handle.createUpdate("INSERT INTO services(" +
                     "id, custom_branding, " +
                     "merchant_name, merchant_telephone_number, merchant_address_line1, merchant_address_line2, merchant_address_city, " +
                     "merchant_address_postcode, merchant_address_country, merchant_email, external_id) " +
                     "VALUES (:id, :customBranding, :merchantName, :merchantTelephoneNumber, :merchantAddressLine1, :merchantAddressLine2, " +
                     ":merchantAddressCity, :merchantAddressPostcode, :merchantAddressCountry, :merchantEmail, :externalId)")
                     .bind("id", service.getId())
-                    .bind("customBranding", customBranding)
+                    .bindBySqlType("customBranding", customBranding, OTHER)
```

Comment:
I looked into https://jdbi.org/#_binding_arguments
I'm just wondering whether wouldn't it be clearer to have something like (copied from docs):
```java
QualifiedType<String> json = QualifiedType.of(String.class).with(Json.class);
...
.bindByType("customBranding", customBranding, json);
```

Reply:
We will have to create new JsonArgumentFactory in order to use qualified type and handle string to json conversion within factory. Agree that it looks clearer but it is more setup for tests

```
@Json
public class JsonArgumentFactory extends AbstractArgumentFactory<String> {
    @Override
    protected Argument build(String value, ConfigRegistry config) {
        // do something specifically for json data
    }
}
```